Tests/Resources/Kernel/Lti1p3TestKernel.php

```diff
@@ -35,13 +35,17 @@ class Lti1p3TestKernel extends Kernel
 {
     use MicroKernelTrait;
 
-    public function registerBundles()
+    public function registerBundles(): iterable
     {
-        return [
+        $bundles = [
             new FrameworkBundle(),
             new SecurityBundle(),
             new Lti1p3Bundle()
         ];
+
+        foreach ($bundles as $class) {
+            yield new $class();
```

Comment:
as i see `$bundles` already contains instantiated objects, i can't understand profit of this changes

Reply:
we are talking here about a TEST kernel :) 

but let's see, did you actually benchmarked this if it really provide perfs improvements ? I'm curious